app/_locales/en/messages.json

```diff
@@ -1,14 +1,14 @@
 {
   "appName": {
-    "message": "Behave!",
+    "message": "Behave! — Offensive Comments Blocker",
```

Comment:
I see, and you already have a longer description.... two more (shoter) ideas... 
"Behave! - offensive comments masked by emoji" or 
"Behave! - toxic comments hidden by emoji"